### DISCUSSION TITLE
3/29(수) 익스텐션

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#40] [General]
익스텐션은 코드의 가독성을 높여주고 나중에 기존 정의되 있는 함수나 변수의 커스텀 역활을 할 수 있습니다. 물론 나중에 많이 사용하기 때문에 언제가는 익히겠지만 지금 빨리 알아두면 사전적 정의와 정확한 사용 방법을 알 수 있기 때문에 한번 공부할 때 확실히 해두는 것을 추천합니다. 🔥

**Comment by Eunho0922** [comment]
넵
